website/docs/migration-guides/yew/from-0_19_0-to-0_20_0.mdx

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
I realized this page is not added to the navigation bar. Could you add it?

Reply:
I think the migration guide should remain "hidden" until released, shouldn't it?

(This is the same as the yew-agent 0.1.0 -> 0.2.0 guide, which is hidden.)

Reply:
Showing it would make 0.19 -> next transition easier but it's something we can do later.

Reply:
you guys are right